test/fail_compilation/ice20324.d

```diff
@@ -0,0 +1,19 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/ice20324.d(14): Error: argument `Foo()` to __traits(getUnitTests) must be a module or aggregate
+fail_compilation/ice20324.d(15): Error: argument `Bar(T)` to __traits(getUnitTests) must be a module or aggregate
+---
+*/
+module ice20324;
+
+template Foo() { unittest{} }
+class Bar(T)   { unittest{} }
+
+// template decls rejected
+alias fooTests = __traits(getUnitTests, Foo);
+alias barTests = __traits(getUnitTests, Bar);
+
+// template instances accepted
+static assert(__traits(getUnitTests, Foo!()).length == 1);
```

Comment:
This should be in a "compiles" test, for some reason it passes locally when testing the single file but not when running the whole test suite.

Reply:
Mmmmh just the -unittest switch missing actually